rfc/0056-sdk3-query.md

```diff
@@ -103,6 +103,11 @@ QueryResult Query(string statement, [QueryOptions options]);
     * Specifies that the query engine should preserve expiration values set on any documents modified by this query.
     * Sent within the JSON Payload as `preserve_expiry` as a JSON boolean
     * Should be added at API stability level of uncommitted.
+  * useReplica(bool) = undefined
+    * Specifies that the query engine should use replica nodes for kv fetches if the active node is down.
+    * Sent within the JSON payload as `use_replica` as the JSON string `"on"` or `"off"`, only if set by the user.
+    * If the SDK does not have the ability to differentiate between not set and `false` then an enum type should be used instead.
```

Comment:
This logic is slightly different than we've otherwise done in the SDKs before.  Typically we would align the server-default with the SDK-default to enable us to align the defaults and avoid needing to turn a boolean value into a tri-state enumeration.

Reply:
After talking with @chvck , it turns out this is in line with our precedence from other similar requirements in the SDK.